demos/smart_classroom_demo_gapi/src/person_action_detector.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "person_action_detector.hpp"
+
+
+#define SSD_LOCATION_RECORD_SIZE 4
+#define SSD_PRIORBOX_RECORD_SIZE 4
+#define NUM_DETECTION_CLASSES 2
+#define POSITIVE_DETECTION_IDX 1
+#define NUM_CANDIDATES 4300
+#define INVALID_TOP_K_IDX -1
+
+void ActionDetector::GetPostProcResult(const float* local_data,
```

Comment:
`fetchResults()` from OMZ is our `GAPI_OCV_KERNEL(OCVPersonDetActionRec...)`, 
`GetDetections` from OMZ is our `GetPostProcResult`.